src/rdkafka_msgset.h

```diff
@@ -29,6 +29,13 @@
 #ifndef _RDKAFKA_MSGSET_H_
 #define _RDKAFKA_MSGSET_H_
 
+typedef struct rd_kafka_aborted_txn_start_offsets_s {
+        rd_avl_node_t avl_node;
+        int64_t pid;
+        int32_t offsets_idx;
```

Comment:
should be int as it points to an rd_list index.

Reply:
now you point it out, I'm thinking `size_t` is probably most ideal here, but `rd_list_elem` takes an `int` so that is most appropriate, yes.